parsa-rajabi-ubc/Home-Together-Canada
# HTC-774/messaging connect backend

# [#774 ](https://github.com/rachellegelden/Home-Together-Canada/issues/774)

## Summary
Connect backend to frontend.

## Relevant Motivation & Context
We need to connect backend to frontend. 
Note: 
- It is not a real time messaging system, you have to go to another nav then come back to see the new messages.
- After send a message, the text area is not empty, will have the value message content that you just send.

## Testing Instructions
1. Create 2 users (userA and userB) in post postman 
2. In postman login as 1 userA , and create some messages that send to userB
3. In post man logout as userA, login as userB, and create some messages that send to userA
4. login in as userA in http://localhost:3002/, then go to messaging
5. Clicks on the conversation card to go in to the conversation detail between 2 users
6. View and check the output
7. Create a message in textarea, and hit send button
8. Go to main page or other nav then come back to messaing, the message you just sent should display in both "converation card" and "conversation detail"
9. Logout as userA in http://localhost:3002/, then login as userB,
10. delete userB account, re-login as userA
11. Go to messaging and trying to send the message to userB again it will show the alert of that user account is deleted.

## Developer checklist prior to opening this pull request:

## Reviewer
